mrmustard/physics/fock.py

```diff
@@ -123,6 +125,37 @@ def ket_to_dm(ket: Tensor) -> Tensor:
     return math.outer(ket, math.conj(ket))
 
 
+def dm_to_ket(dm: Tensor) -> Tensor:
+    r"""Maps a density matrix to a ket if the state is pure.
+
+    If the state is pure :math:`\hat \rho= |\psi\rangle\langle \psi|` then the
+    ket is the eigenvector of ``\rho`` corresponding to the eigenvalue 1.
+
+    Args:
+        dm: the density matrix
+
+    Returns:
+        Tensor: the ket
+    """
```

Comment:
(minor) missing raises
```suggestion
    Returns:
        Tensor: the ket
        
    Raises:
        ValueError: if ket for mixed states cannot be calculated
    """
```